pygeo/constraints/baseConstraint.py

```diff
@@ -55,7 +57,7 @@ def evalFunctionsSens(self, funcsSens, config):
         funcsSens : dict
             Dictionary to place function values
         """
-        pass
+        return
```

Comment:
I don't understand the reason for the change here. Is it related to the decorator?

Reply:
Hmm I forget exactly why I made this change---I'll change it back.

Reply:
I remember now: this is because pylint would throw "unnecessary pass statement" warning: https://vald-phoenix.github.io/pylint-errors/plerr/errors/basic/W0107.html but I guess the proper way is to use the docstring as the main part of the code, and simply not put `pass`. It feels weirdly unreadable though. In any case this is something we can figure out later.

Reply:
Yeah, not a problem for me. I was just curious.